src/main/java/com/iexec/common/sms/SmsConfig.java

```diff
@@ -0,0 +1,6 @@
+package com.iexec.common.sms;
+
+public class SmsConfig {
+
+    public static final String DOMAIN = "IEXEC_SMS_DOMAIN"; // TODO: Add session salt after domain
```

Comment:
DOMAIN should depends on SMS version, e.g IEXEC_SMS_DOMAIN_v1.0.0, I don't think we can move it to common (or we can call it DOMAIN_PREFIX)

Reply:
Renaming it does not help, so I'll leave it in the SMS.